qutebrowser/mainwindow/prompt.py

```diff
@@ -635,6 +641,9 @@ def _insert_path(self, index, *, clicked=True):
             clicked: Whether the element was clicked.
         """
         path = os.path.normpath(self._file_model.filePath(index))
+        if index == QModelIndex():
+            path = os.path.join(self._file_model.rootPath(), self._to_complete)
```

Comment:
This is a bit confusing to read - can you please move the `path` assignment before this one to an `else:` instead?